### PR TITLE
Unsafe merge with outdated eng, check carefully!!!

### PR DESCRIPTION
**Pull request by MasterCoookie**
merged + fixed filenames

### CODE REVIEW

**Review by Direckton — APPROVED** [review]

File: public/scripts/comment-send-script.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ function sendComment(postID) {
 	const commentForm = document.getElementById('comment-form');
 	const formData = new FormData(commentForm);
 	formData.append('postID', postID);
+	console.log(postID);
```
Comment (tomazaw) [inline public/scripts/comment-send-script.js]: Wywal to

File: views/post/postView.ejs [hunk 2]
```diff
@@ -194,6 +198,7 @@
     <script>Array.from(document.getElementsByClassName('reply-to-comment')).forEach(
             (element) => {
                 element.addEventListener('click', (event) => {
+                    console.log(event.target.form.id);
```
Comment (tomazaw) [inline views/post/postView.ejs]: same thing as earlier

**Review by tomazaw — CHANGES_REQUESTED** [review]

File: public/scripts/comment-send-script.js (hunk 1 above)
Comment (macibol775) [inline public/scripts/comment-send-script.js]: why?

File: views/post/postView.ejs (hunk 2 above)
Comment (macibol775) [inline views/post/postView.ejs]: why?

**Review by macibol775 — CHANGES_REQUESTED** [review]
2 console.logs can be deleted, 
STOP USING F*CKING PREETIER

**Review by tomazaw — APPROVED** [review]

**Pull request merged**